demo/helm/rook-operator/README.md

```diff
@@ -15,7 +15,7 @@ Alternatively, to deploy from a local checkout of the rook codebase (until the r
 
 ```console
 $ cd demo/helm/rook-operator
-$ helm install --name rook-operator --namespace rook .
+$ helm install --name rook-operator --namespace default .
```

Comment:
this should just be the namespace where the operator starts. It shouldn't matter in what namespace it starts though.